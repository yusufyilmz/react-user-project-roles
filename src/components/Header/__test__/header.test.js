

import React from 'react';
import {Header} from '../index';
import { shallow } from 'enzyme'


describe('HEADER COMPONENT', () => {

    let message, wrapper;

    beforeEach(() => {
        message = 'test';
        wrapper = shallow(
            <Header message={message} />
        );
    });
   
    test('renders element correctly', () => {
        expect(wrapper.find('div').text()).toBe(message);
    });

    test('snapchat correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
})
