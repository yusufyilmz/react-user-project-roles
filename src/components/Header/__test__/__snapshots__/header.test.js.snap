// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component snapchat correctly 1`] = `
<div>
  <h1>
    test
  </h1>
</div>
`;
